### DISCUSSION TITLE
Create discussion after release has been created

### DISCUSSION THREAD

**Discussion by Jericho** [community/community#3922] [Discussions]
I created a release and forgot to check the "Create a discussion for this release" option. Is there a way to create the discussion and link it to the release after the fact, or is my only option to delete the release and re-create it?

**Comment by Andre601** [comment]
You could just create the discussion manually and add the link to the release somewhere in it.
After all is this everything the option in the release does...

**Reply by Jericho** [reply]
Not quite. When you check the "create discussion" checkbox, GitHub adds this to the release:

![image](https://user-images.githubusercontent.com/112710/117666637-10bad580-b172-11eb-8541-6d6426982460.png)

which you do not get if you manually create the discussion.

**Reply by Andre601** [reply]
That seems to be new as I never saw this before.

**Comment by stianjensen** [comment]
Second this. This missing possibility also makes it hard to now adopt discussions as we cannot easily set it up for the releases that are already in progress.

**Comment by mkurz** [comment]
Highly needed. I forgot to activate the checkbox and would love to add a discussion after the release was published.
Thanks!

**Reply by kevinbluer** [reply]
Agreed, just had the same situation...this would be a nice addition!

**Comment by morpheaux** [comment]
Agreed. A great addition!

**Comment by Morsmalleo** [comment]
Agreed, done the same thing, forgot to check the box to create a discussion for said release, so this would be nice option to have.

**Comment by HareshPrajapati** [comment]
As of my last knowledge update in September 2021, GitHub does not provide a direct way to retroactively create a discussion and link it to a release that was created without the "Create a discussion for this release" option selected. However, deleting and recreating the release might not be necessary. You have a few alternative options to consider:

1. **Create a New Discussion:** You can manually create a new discussion related to the release. While it won't be directly linked to the release in GitHub's interface, you can include information about the release in the discussion post, such as the version number, changes, and relevant details.

2. **Edit Release Notes:** If you want to add details or information related to the release, you can edit the release notes of the existing release to include the necessary information. However, this won't create a discussion thread.

3. **Use Release Description:** If you're using the release description field when creating a release, you can include a link to a separate discussion thread related to the release.

4. **Feature Request:** If this feature is important to your workflow, you can consider submitting a feature request to GitHub through their official channels. They might consider adding the ability to link a discussion to a release after it's created.

Before taking any action, it's advisable to consult GitHub's official documentation or support resources to see if there have been any updates or changes since my last update in September 2021. Additionally, you might find discussions or solutions within the GitHub community forums related to this topic.

**Comment by mkurz** [comment]
Instead of "Create discussion after release has been created" I would suggest to be able to just connect a release with a discussion, which then would solve this problems. Here are some mocks I came up with so you know what I mean:
<hr>
<kbd><img width="400" src="https://github.com/user-attachments/assets/88c3626e-56a8-4128-b021-24a274805ab5" /></kbd>
<hr>
<b>To choose which discussion to link:</b>
<br>
<kbd><img width="400" src="https://github.com/user-attachments/assets/58790579-775e-40a6-b190-2ade50a862cc" /></kbd>

<br><br>
Of course there should be an option to "clear" the discussion which unsets it, but I didn't mock this :wink:

**Comment by Christoph142** [comment]
Here we are in 2026 and it's still not possible. I'm actually 99% sure I checked the box when I drafted the release, but it's still missing after publishing the draft and there's no way to connect it.
